cmd/mr_show.go

```diff
@@ -98,7 +106,43 @@ WebURL: %s
 		mr.Author.Username, milestone, labels, mr.WebURL)
 }
 
-func printMRDiscussions(discussions []*gitlab.Discussion) {
+func printMRDiscussions(discussions []*gitlab.Discussion, since string, mrNum int) {
+	NewAccessTime := time.Now().UTC()
+	viper.Reset()
+	mrEntry := fmt.Sprintf("mr%d", mrNum)
+	viper.SetConfigName("labmetadata")
+	viper.SetConfigType("hcl")
+	viper.AddConfigPath(".git")
+	// write data
+	if _, ok := viper.ReadInConfig().(viper.ConfigFileNotFoundError); ok {
+		if err := viper.WriteConfigAs(".git/labmetadata.hcl"); err != nil {
```

Comment:
As mentioned in other comment, doesn't need to be hcl, but specifically here, I'm on board to put files in `.git` (particularly since these are ignored by git by default and being local to the repo solves a lot of problems), but I'd like to talk a bit more up front on some naming conventions.

First I'd like to go with either a `lab_` preview to file names or place files in a `.git/lab/` directory (leaning towards the later). Second, does `metadata` make sense to describe whats being stored in this file? It's certainly metadata, but it's also specific to `lab issue/mr show` operations, so something describing that seems preferred. Perhaps `.git/lab/show_op_log` or `.git/lab/cmd_show_metadata` or `.git/lab/show_metadata` (with w/e extension)

Reply:
I like the last option.  .git/lab/show_metadata sounds good to me.